lib/plottr_firebase/migrations/copy-known-file-information-to-authorisations.js

```diff
@@ -23,14 +25,92 @@ if (!admin.apps.length) {
   }
 }
 
+const RUN_DATE = new Date()
+
 const main = (argv) => {
   const rl = readline.createInterface({ input: stdin, output: stdout })
   const userId = argv[2]
-  console.log('User id: ' + userId)
+  if (userId) {
+    console.log('Running for user id: ' + userId)
+  } else {
+    console.log('No user supplied running for all users.')
+  }
 
-  rl.question('Hit enter to continue...', () => {
-    markDeletedAuthorisedDocumentsAsDeleted(userId)
-  })
+  rl.question(
+    'Is this the production run?\n(Only "Yes" -case sensitive- will be interpreted as affirmative): ',
+    (response) => {
+      const readOnlyMode = response !== 'Yes'
+      console.log('Read only mode: ', readOnlyMode)
+      rl.question('Please enter your userId: ', (executingUserId) => {
+        checkUserId(executingUserId).then((proceed) => {
+          if (proceed) {
+            console.log('User exists.  Executing script.')
+            const userIds = userId ? Promise.resolve([userId]) : allUserIds()
+            userIds
+              .then((uids) => {
+                console.log(`Will run script for users with ids: ${JSON.stringify(uids, null, 2)}`)
+                return sequencePromises(
+                  uids.map((uid) => {
+                    return () => {
+                      console.log(`> ${uid}`)
+                      return markDeletedAuthorisedDocumentsAsDeleted(
```

Comment:
I think this migration is doing the wrong thing. The name of the file suggests it should be the  copy-known-file-info-to-authorisations one